shared/ui/utils/e2eTag.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+// TODO: needs to be environment-aware
+
+export const e2eTag = (value, tag, tagValue) => {
```

Comment:
There must be a less overt method for tagging our code in a useful way. Why not just use a class or an ID that is names the item(s) in a functional but not test-focused manner?